### PR TITLE
Removed clutter files; Added deck practicing

### PR DESCRIPTION
**Pull request by MartynasJonkus**

### CODE REVIEW

**Review by d-pilipavicius — APPROVED** [review]

File: aiKart/ClientApp/src/components/deck/CardView.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React, { useState, useEffect } from 'react'
+import { Card, CardBody, CardTitle, CardText } from "reactstrap";
+
+const CardView = (props) => {
```
Comment (rokas-migauskas) [inline aiKart/ClientApp/src/components/deck/CardView.js:4]: Naudojam mes react redux, tai props nereiketu passint, galima extra idet metoda i reducer settint dabartine korta

File: aiKart/ClientApp/src/components/deck/CardView.js
Comment (rokas-migauskas) [inline aiKart/ClientApp/src/components/deck/CardView.js:1]: Hoverinant kortele cursor i click pakeist gal? Arba atskira mygtuka uzdet revealinimui

Comment (rokas-migauskas) [inline aiKart/ClientApp/src/components/deck/CardView.js:1]: Direktorija /card geriau butu, o ne /deck

Comment (MartynasJonkus) [reply aiKart/ClientApp/src/components/deck/CardView.js:1]: Done, pridejau pointer mouse ant hover

Comment (MartynasJonkus) [reply aiKart/ClientApp/src/components/deck/CardView.js:1]: Done

**Pull request merged**